### PR TITLE
doc/cephadm: Co-location of daemons

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Signed-off-by: Sebastian Wagner <sewagner@redhat.com>

#39931

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by anthonyeleven** [comment]
Is `count-per-host` a new feature for Quincy?  I don't know if we have a policy or even convention, but I personally try to note in which release functionality is first found, so as to not confuse people running older releases.  Granted one can browse former doc releases, but I like to go the extra step.

File: doc/cephadm/services/index.rst [hunk 1]
```diff
@@ -402,6 +402,30 @@ YAML can also be used to specify limits on hosts:
         - host2
         - host3
 
+.. _cephadm_co_location:
+
+Co-location of daemons
+----------------------
+
+Cephadm supports the deployment of multiple daemons on the same host:
```
Comment (adk3798) [inline doc/cephadm/services/index.rst:410]:
```suggestion
Cephadm supports the deployment of multiple daemons from the same service on the same host:
```
NIT

**Review by adk3798 — APPROVED** [review]
LGTM. Do we also want to mention here that this is how you do upgrade on single host clusters (or maybe link from single host defaults section to here)?

**Comment by anthonyeleven** [comment]
LGTM, thanks for adding the note re Pacific.

**Pull request merged**